server/apps/main/forms.py

```diff
@@ -56,6 +56,12 @@ class ShareExperienceForm(forms.Form):
     moderation_status = forms.ChoiceField(choices = statuses, widget = forms.Select(), required=False)
     moderation_status.group = "hidden"
 
+    moderation_comments = forms.CharField(label='Moderator comments', max_length=500, strip=True, required=False,
```

Comment:
Do we actually need the moderation_comments in this form?

Reply:
No we don't thought I had commented that out!

Reply:
Ah, and I see that currently we don't actually use the moderation form! `model_to_form` still uses the `ShareExperienceForm`, which explains why it actually showed up! :D

Reply:
Yeah I've just changed that